_benchmark/user-guide/finetine-workloads.md

```diff
@@ -0,0 +1,55 @@
+    2. Take the store size and divide it by the product of the previous step. The value of the multiple is used to set the number of sample docs from the reference index. 
+5. **Target Cluster Configuration**-  Consider the configuration and characteristics the target cluster and factor those into how you generate the workload.
+
+
+## Example
+
+The following example contains an index named `stocks`. The `stocks` index includes documents with stats on all stocks being traded in the New York Stock Exchange (NYSE). Using OpenSearch Dashboards gives information about the `stocks` index, as shown in the following code example:
+
+```
+health status index  uuid              pri rep docs.count docs.deleted store.size pri.store.size
+green  open   stocks asjdkfjacklajldf   12   1  997818020    232823733    720gb    360gb
+```
+
+Using the information, you can start tweaking the workload to your specifications
+
+1. **Fetch queries that are associated with this index:**  Obtain the queries needed to make requests to the stocks index.
+2. **Find shard size for index:** To get the shard size of the index, divide the store size with the number of shards in the index. From this, we get `720 / (12 + (12 * 1)) = 30`. In other words, 30GB is the shard size. You can  verify this by dividing the primary store size value by the primary shards and you will get the same value
+3. **Decide how many shards you want for your index:** Determine the number of shards needed in the index to understand your application. For example, if you want your index to hold 300GB of documents, but 300GB is too much for the benchmark, determine a number that makes a shard amount that makes sense. For example, 300GB of documents divided by the 30GB shard size determined in the last step, or `300 / 30 = 10`, gives a shard of 10. These 10 shards can either be 10 primary shards and 0 replicas, 5 primary shards and 1 replica, or 2 primary shards and 4 replicas. The shard configuration depends on your clusters needs index needs.
+4. **Decide how many docs you want to extract:** To retain 30gb and to have 10 shards, you need to extract at least 300GB worth of documents. To get how many docs this would approximately equal, take the store size value and divide it by the index size value you decide, for this example `720 / 300 = 2.4`. Since we want to make sure we reach a value of 30GB per shard, its best to round down and chose 2 as the extraction multiple, which means OpenSearch Benchmark extracts every other document. From a query perspective, this might be better than grabbing the first half of the corpus, since documents from the entire index are extracted.
```

Comment:
```suggestion
4. **Decide how many docs you want to extract:** To retain 30 GB and have 10 shards, you need to extract at least 300 GB worth of documents. To get the number of documents to extract, take the store size value and divide it by the index size value you decide, in this example`720 / 300 = 2.4`. Since we want to make sure we reach a value of 30 GB per shard, its best to round down and chose 2 as the extraction multiple, which means OpenSearch Benchmark extracts every other document. From a query perspective, this might be better than grabbing the first half of the corpus, since documents from the entire index are extracted.
```